### PR TITLE
Import from Git (Private Repositories) UPDATE

### PR DESCRIPTION
**Pull request by DeRuina**
Added documentation for Import from Git (Private Repositories) using Web GUI and CLI
https://csc-guide-preview.2.rahtiapp.fi/origin/DeRuina-patch-1/cloud/rahti2/images/creating/

## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) of your branch.

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes the automated tests. If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/cloud/rahti2/images/creating.md [hunk 1]
```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
 cat Dockerfile | oc new-build -D -
 ```
 
+### Import from Git (Private Repositories) using the Web GUI
+
+Deploying a private Git repository to Rahti involves setting up the necessary authentication to access your private repository. Without proper authentication, you will see the error "URL is valid but cannot be reached" (seen in the pictures below). Here's how to resolve this using two authentication methods:
```
Comment (lvarin) [inline docs/cloud/rahti2/images/creating.md:180]: Screenshot showing this error should be just below this paragraph.

File: docs/cloud/rahti2/images/creating.md [hunk 2]
```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
...
+     - Navigate to "Developer settings" > "Personal access tokens".
+     - Click on "Generate new token".
+     - Select the scopes you need (typically, you'll need `repo` scope for private repositories).
+     - Generate the token and copy it.
+   
+   - **GitLab:**
+     - Go to your GitLab profile settings.
+     - Navigate to "Access Tokens".
+     - Give your token a name, select the required scopes (e.g., `api`, `read_repository`), and create the token.
+     - Copy the token.
```
Comment (lvarin) [inline docs/cloud/rahti2/images/creating.md:197]: Indentation is not correct. It is not rendered correctly

File: docs/cloud/rahti2/images/creating.md [hunk 3]
```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
...
+
+2. **Add the Token to Rahti:**
+     - Under "Source Secret" choose "Create new Secret"
+     - Name the secret, under "Authentication type" choose "Basic Authentication"
+     - Paste the token and create
```
Comment (lvarin) [inline docs/cloud/rahti2/images/creating.md:202]: Screenshot showing the Create new screen must be under this paragraph.

File: docs/cloud/rahti2/images/creating.md [hunk 4]
```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
...
+
+   - **GitHub:**
+     - Go to your GitHub account settings.
+     - Navigate to "SSH and GPG keys".
+     - Click "New SSH key" and paste the contents of your `id_rsa.pub` file.
+   
+   - **GitLab:**
+     - Go to your GitLab profile settings.
+     - Navigate to "SSH Keys".
+     - Add a new SSH key and paste the contents of your `id_rsa.pub` file.
```
Comment (lvarin) [inline docs/cloud/rahti2/images/creating.md:224]: Same indentation issue

File: docs/cloud/rahti2/images/creating.md [hunk 5]
```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
...
+![fd4c203a-f8e3-4994-b1a4-9a2b4960d4be](https://github.com/user-attachments/assets/ea238069-de97-470f-bb86-03980456eb3a)
+
+![434bf283-c7f5-49d1-8f58-72480f084ce2](https://github.com/user-attachments/assets/1c17a62b-f3b6-404d-8568-24e05e00c1c4)
+
+### Import from Git (Private Repositories) using the CLI
+
+This assumes that the users has generated SSH keys and registered their public key with GitHub.
+
+ 
+**Log into OpenShift CLI (`oc`)**:
```
Comment (lvarin) [inline docs/cloud/rahti2/images/creating.md:242]: Add link to the docs.csc.fi article where the login is explained

File: docs/cloud/rahti2/images/creating.md [hunk 6]
```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
...
+   ```bash
+   oc login <cluster-url>
+   ```
+
+**Create a New Project**:
+   ```bash
+   oc new-project <project-name>
```
Comment (lvarin) [inline docs/cloud/rahti2/images/creating.md:249]: This does not work, it needs csc_project number. Add link to docs.csc.fi where it is explained how to create a new project

File: docs/cloud/rahti2/images/creating.md [hunk 7]
```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
...
+   ```
+
+
+**Deploy the Application**:
+   ```bash
+   oc new-app <repository-url> --name=<application-name>
+   ```
+
+**Monitor the Build**:
+   - monitor logs
```
Comment (lvarin) [inline docs/cloud/rahti2/images/creating.md:269]: Inconsistent indentation

File: docs/cloud/rahti2/images/creating.md (hunk 4 above)
Comment (DeRuina) [reply docs/cloud/rahti2/images/creating.md]: fixed

File: docs/cloud/rahti2/images/creating.md (hunk 5 above)
Comment (DeRuina) [reply docs/cloud/rahti2/images/creating.md]: fixed

**Comment by DeRuina** [comment]
@lvarin Please review

**Pull request merged**